modules/movie.py

```diff
@@ -12,8 +12,9 @@
 """
 
 import json
+import urllib
 import urllib2
```

Comment:
I vote we drop both of these and start migrating stuff like this to urllib3 while we're making changes. Opened an "issue", which will be marked as enhancement, here: https://github.com/myano/jenni/issues/135